ci-operator/populate-secrets-from-bitwarden.sh

```diff
@@ -16,6 +16,25 @@ set -o xtrace
 #          content of the BitWarden vault after your current session was started,
 #          you will need to create a new session to be able to view those changes.
 
+readonly CURRENT_OC_CONTEXT=$(oc config current-context)
+readonly OC_PROJECT=$(echo ${CURRENT_OC_CONTEXT} | cut -d "/" -f1)
+readonly OC_CLUSTER=$(echo ${CURRENT_OC_CONTEXT} | cut -d "/" -f2)
+readonly OC_USER=$(echo ${CURRENT_OC_CONTEXT} | cut -d "/" -f3)
```

Comment:
`readonly` masks errors in the assignment:

```sh
$ readonly f=$(false); echo $?
0
```

Reply:
Done with the help [here](https://github.com/koalaman/shellcheck/issues/558#issuecomment-248772599).